markdown/events/2023/hackathon-march-2023/index.md

```diff
@@ -97,9 +97,103 @@ Why stop at only sharing modules? [Subworkflows](https://github.com/nf-core/modu
+
+<div class="table-responsive">
+    <table class="table table-hover table-sm table-bordered">
+        <thead>
+            <tr>
+                <th>Time</th>
+                <th>Mon. 27 Mar., 2023</th>
+                <th>Tue. 28 Mar., 2023</th>
+                <th>Wed. 29 Mar., 2023</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td data-timestamp="1679875200" data-timeformat="HH:mm z">02:00</td>
+                <td background-color:navy; rowspan="8"></td>
+                <td background-color:navy; rowspan="14">Hack!</td>
+                <td background-color:navy; rowspan="14">Hack!</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679878800" data-timeformat="HH:mm z">03:00</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679882400" data-timeformat="HH:mm z">04:00</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679886000" data-timeformat="HH:mm z">05:00</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679889600" data-timeformat="HH:mm z">06:00</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679893200" data-timeformat="HH:mm z">07:00</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679896800" data-timeformat="HH:mm z">08:00</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679900400" data-timeformat="HH:mm z">09:00</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679904000" data-timeformat="HH:mm z">10:00</td>
+                <td>Welcome<br>Location: Lecture Theatre</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679907600" data-timeformat="HH:mm z">11:00</td>
+                <td rowspan="5">Hack!</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679914800" data-timeformat="HH:mm z">12:00</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679914800" data-timeformat="HH:mm z">13:00</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679918400" data-timeformat="HH:mm z">14:00</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679922000" data-timeformat="HH:mm z">15:00</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679925600"  data-timeformat="HH:mm z">16:00</td>
+                <td>Daily sync<br>Location: Lecture Theatre</td>
+                <td>Daily sync<br>Location: Lecture Theatre</td>
+                <td>Final wrap-up<br>Location: Lecture Theatre</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679929200"  data-timeformat="HH:mm z">17:00</td>
+                <td rowspan="6">Hack!</td>
+                <td rowspan="6">Hack!</td>
+                <td rowspan="6"></td>
```

Comment:
I think it will be confusing for Europeans to work that long 🙁 
Also 16:00 is prime child pick-up time I think (@jfy133 ?), can we make it 15:00 or is that too early for US west-coast?

Reply:
I'm not sure if there will be a perfect time.
It was 18:00 at the Summit and 17:00 in March 2022.
16:00 CET = 07:00 PST